extensions/amp-next-page/amp-next-page.md

```diff
@@ -82,10 +82,14 @@ usually be placed directly after the unique content of a given page: for
 example, at the end of a news article or recipe, but before the footer or
 other content repeated across articles.
 
-{% call callout('Note', type='note') %} For performance reasons the
-component will render a maximum of three documents (total) on screen at one
-time. This limit may be changed or removed in the future.
-{% endcall %}
+The component will render a total maximum of three documents on screen in a
+single instance. This limit may be changed or removed in the future. 
+
+[tip type="important]
+**Important**
```

Comment:
Hi @CrystalOnScript I don't see the link to the issue here. Am I missing anything?